cwms/timeseries/critscript.py

```diff
@@ -0,0 +1,79 @@
+import re
+
+import pandas as pd
+
+import cwms
+
+
+def crit_script(file_path, office_id, group_id):
+    def parse_crit_file(file_path):
+        """
+        Parses a .crit file into a dictionary. Parses for the timeseries ID and Alias
+
+        :param file_path: Path to the .crit file
+        :return: A dictionary containing the parsed key-value pairs
+        """
+        parsed_data = []
+        with open(file_path, "r") as file:
+            for line in file:
+                # Ignore comment lines and empty lines
+                if line.startswith("#") or not line.strip():
+                    continue
+
+                # Extract alias, timeseries ID, and TZ
+                match = re.match(r"([^=]+)=([^;]+);(.+)", line.strip())
+                if match:
+                    alias = match.group(1).strip()
+                    timeseries_id = match.group(2).strip()
+                    alias2 = match.group(3).strip()
+
+                    parsed_data.append(
+                        {
+                            "Alias": alias + " " + alias2,
```

Comment:
I just storing a timeseries into a timeseries group and the deliminator can't be a space.  That won't save into the alias field.  use a : instead here.  alias + ":" + alias2

also need the case if they don't have match.group(3).  they might just have alias and timeseries_id.   could set alias = match.group(1).strip() then if len(match) = 3 alias = alias + ":" + match.group(3).strip() or something like that.